### DISCUSSION TITLE
Adding Theory about the play.

### DISCUSSION THREAD

**Discussion by Abhishek-90** [reactplay/react-play#184] [Ideas]
For every play here, there is a Related Readme file present where we add basic information about the play and concepts we use in it. I suggest we add a Link to this readme file in our plays. This link we direct us to a page where we can find the theory of our Readme file. This will help the users/new react developers to understand the workings of the play better. 

We can also add basic theory about the concepts we use inside this play.

**Comment by atapas** [comment]
Hey @Abhishek-90 Thanks for putting this idea. Can I toss in something?

Each of the plays has a way to link to their blog(if any) optionally. How about we do the code changes to link to this Readme file of the play if the creator hasn't supplied a blog URL for the play? That way, the click of the icon will take them to the readme. What do you think?

![image](https://user-images.githubusercontent.com/3633137/168552070-e64f1516-c77d-4cf0-bc6c-6f3417b20488.png)

Only one catch I have is, clicking on the GitHub icon takes the user to the respective code that anyway lands into the readme.

**Reply by Abhishek-90** [reply]
This sounds like a good idea, blog or readme file both serves the purpose of putting a theory in front of user. A simple code change might select either blog or readme file with Blog with a preference.

**Reply by atapas** [reply]
Created an issue: https://github.com/atapas/react-play/issues/200

Closing the discussion.
